db/migrate/20210810220944_create_students.rb

```diff
@@ -0,0 +1,12 @@
+class CreateStudents < ActiveRecord::Migration[6.1]
+  def change
+    create_table :students do |t|
+      t.string :name
+      t.string :lastname
+      t.string :email
+      t.string :password_digest
```

Comment:
Do you have a user-type table where these parameters already exist?

Reply:
we don´t have it